packages/e2e-tests/src/specs/editor/floatingMenu.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { createNewStory } from '@web-stories-wp/e2e-test-utils';
+
+describe('Floating Menu', () => {
+  const floatingMenu = 'div[role="menu"]';
```

Comment:
🤔 This might not be specific enough. There's more than 1 menu role present on the canvas. 

I think looking for the section with the associated aria label for the floating menu might be more reliable. The screenshot I attached is querying the role menu vs section with aria label. Thoughts? 


<img width="575" alt="Screen Shot 2022-04-18 at 1 29 14 PM" src="https://user-images.githubusercontent.com/10720454/163872989-13e0dd41-af1d-4c1c-87da-f9d0f1544855.png">

Reply:
@mariana-k I also suggest locating that element by the `Design menu` label (like in my first comment). We already have labels (we need them for a11y) so let's use them whenever possible (they point to very specific things, so we can avoid issues like this^).

Reply:
Done